helpers/rpc.ts

```diff
@@ -0,0 +1,57 @@
+import axios, { AxiosRequestConfig, HttpStatusCode } from "axios"
+
+// http 400s, rate limits
+const CooldownOnClientError = 60 // 1 minute
+
+// http 500x, server side error
+const CooldownOnServerError = 300 // 5 minute
+
+// undefined, 0 or a past time is a ready to call endpoint
+const GlobalEndpointsCooldown: { [key: string]: number } = {}
+
+async function sleep(time: number) {
+  return new Promise((resolve) => setTimeout(resolve, time))
```

Comment:
setTimeout uses millisecond format iirc, so need to either *1000 here or send in ms in calls